probe-rs/src/config/flash_algorithm.rs

```diff
@@ -1,9 +1,9 @@
-use super::flash_properties::FlashProperties;
-use super::memory::{PageInfo, RamRegion, SectorInfo};
+use super::{FlashProperties, PageInfo, RamRegion, SectorInfo};
 use crate::core::Architecture;
 use crate::flashing::FlashError;
 use crate::{architecture::riscv, Target};
-use std::{borrow::Cow, convert::TryInto};
+use std::borrow::Cow;
```

Comment:
I think this is unused :)

Reply:
It's used in tests. For some reason rust-analyzer complains anyway. Moved the imports to the tests.